src/games/strategy/triplea/oddsCalculator/ta/IOddsCalculator.java

```diff
@@ -52,5 +52,6 @@ AggregateResults setCalculateDataAndCalculate(final PlayerID attacker, final Pla
 
   void addOddsCalculatorListener(final OddsCalculatorListener listener);
 
+  // TODO: this method appears to never used.
```

Comment:
IE: such a change should be in its own PR, it is completely independent of this work.